test/els_references_SUITE.erl

```diff
@@ -148,6 +149,24 @@ export_entry(Config) ->
   assert_locations(Locations, ExpectedLocations),
   ok.
 
+%% Issue #245
+-spec flush_references_db(config()) -> ok.
+flush_references_db(_Config) ->
```

Comment:
Nit: Maybe this test case should be named `purge_references` since that's what the new feature is doing, and it's not so related to the DB itself IMHO.